packages/grid/x-data-grid-premium/src/tests/cellSelection.DataGridPremium.test.tsx

```diff
@@ -0,0 +1,296 @@
+import * as React from 'react';
+import { spy } from 'sinon';
+import { expect } from 'chai';
+import { getCell } from 'test/utils/helperFn';
+// @ts-ignore Remove once the test utils are typed
+import { createRenderer, fireEvent, act } from '@mui/monorepo/test/utils';
+import {
+  DataGridPremium,
+  DataGridPremiumProps,
+  GridApi,
+  useGridApiRef,
+  gridClasses,
+} from '@mui/x-data-grid-premium';
+import { getBasicGridData } from '@mui/x-data-grid-generator';
+
+describe('<DataGridPremium /> - Cell Selection', () => {
```

Comment:
Sentence case, I think in this case.

```suggestion
describe('<DataGridPremium /> - Cell selection', () => {
```

Reply:
My thought is that it seems that can apply https://www.notion.so/mui-org/Style-guide-2a957a4168a54d47b14bae026d06a24b#d8e9cfb2423c4f83844a9f29d2032e3f in these cases.